src/javascript/app/pages/user/account/settings/two_factor_authentication.js

```diff
@@ -0,0 +1,116 @@
+const QRCode           = require('davidshimjs-qrcodejs');
+const Client           = require('../../../../base/client');
+const BinarySocket     = require('../../../../base/socket');
+const FormManager      = require('../../../../common/form_manager');
+const getPropertyValue = require('../../../../../_common/utility').getPropertyValue;
+const localize         = require('../../../../../_common/localize').localize;
+
+const TwoFactorAuthentication = (() => {
+    const form_id = '#frm_two_factor_auth';
+    const state = ['disabled', 'enabled'];
+    let current_state,
+        next_state,
+        qrcode; // eslint-disable-line
```

Comment:
what was eslint giving error on? can we disable that exact rule instead of all eslint?

Reply:
Thanks will add the specific rule. eslint is giving error for no-unused-vars because QR code is added like this to the DOM:
```
        qrcode = new QRCode(document.getElementById('qrcode'), {
            text,
            width       : 160,
            height      : 160,
            correctLevel: QRCode.CorrectLevel.H,
        });
```